THIRD-PARTY-LICENSES.txt

```diff
@@ -55,3 +55,29 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
+------
+
+** libcbor; version 0.11.0 -- https://github.com/PJK/libcbor
```

Comment:
can we omit the version here?

THIRD-PARTY-LICENSES.txt isn't being updated by import_libcbor.py
but even if it were, we also have that scallion tool that other people will use in the future to add more licenses to this file, and we're never going to remember to update that as well...

or is it scallion that put this in here to begin with?